deployment/infra/k8s/jaeger-loadbalancer.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service                    # Type of kubernetes resource
+metadata:
+  name: jaeger-loadbalancer   # Name of the resource
+  annotations:
```

Comment:
I think the hashicorp consul annotation should be removed as we don't have consul (and it may cause problems)